L1Trigger/L1THGCal/plugins/be_algorithms/SingleCellClusterAlgo.cc

```diff
@@ -32,14 +35,24 @@ class SingleCellClusterAlgo : public Algorithm<HGCalTriggerCellBestChoiceCodec>
         }
 
     private:
+        
+        edm::ParameterSet geometry_config_;
+        std::string geometry_name_;
```

Comment:
If these two variables `geometry_name_` and `geometry_config_` are not use, remove them